docs/function-instructions.md

```diff
@@ -366,6 +376,7 @@ If more than five books are present, only the top five should be returned.
 ```javascript
 mostPopularBooks(books);
```

Comment:
To be consistent with how we are naming the other functions (i.e., starting with a verb), maybe we can call this function `getMostPopularBooks()`?